packages/blockchain-wallet-v4-frontend/src/data/components/swap/sagas.ts

```diff
@@ -196,19 +196,7 @@ export default ({ api, coreSagas, networks }: { api: APIType; coreSagas; network
       const payment = paymentGetOrElse(BASE.coin, paymentR)
       if (onChain) {
         try {
-          const hotWalletAddress = selectors.core.walletOptions
-            .getHotWalletAddresses(yield select(), Product.SWAP)
-            .getOrFail('No extra data') as String
-          if (typeof hotWalletAddress !== 'string') {
-            throw new Error('Hot Wallet Address not located.')
-          }
-          yield call(
-            buildAndPublishPayment,
-            payment.coin,
-            payment,
-            order.kind.depositAddress,
-            hotWalletAddress
-          )
+          yield call(buildAndPublishPayment, payment.coin, payment, order.kind.depositAddress)
```

Comment:
i didn't make this change?

Reply:
probably because your branch is out of sync with release/4.62 branch.  address merge conflicts and this should resolve